pythonBasics3.py

```diff
@@ -7,33 +7,36 @@
 # import regular expression module
 import re
 
-# # Part A. starts_with_number
-# Define a function ends_with_number(s) that takes a string and returns true
-# if it ends with a number and false otherwise.
+# # Part A. starts_with_non_number
+# Define a function starts_with_non_number(s) that takes a string and returns true
+# if it starts with anything that isn't a number and false otherwise.
 # (For our purposes, a number is any character that is 0,1,2,3,4,5,6,7,8, or 9.)
 # Note: Be sure to use RegEx!
-def ends_with_number(s):
+def starts_with_non_number(s):
```

Comment:
Maybe name func does_not_start_with_number

Reply:
I thought about using that but I didn't want to make it too long. It's not great as it is, but I didn't want it longer. It also hints at the solution of using character classes

Reply:
This'll solve both, `starts_without_positive_num`

Reply:
Unless Dr. Najjar wants me to change it, I will just make it clear in the instructions that the first character must be a number

Reply:
I'd go with starts_with_non_number. It is clear that way what the function needs to do.